src/api/Main/BuisnessLayer/Shrooms.Domain/Services/Kudos/KudosService.cs

```diff
@@ -41,6 +41,7 @@ public class KudosService : IKudosService
         private readonly IDbSet<ApplicationUser> _usersDbSet;
         private readonly IRepository<KudosLog> _kudosLogRepository;
         private readonly IRepository<ApplicationUser> _applicationUserRepository;
+        private readonly IDbSet<Organization> _organizationsDbSet;
```

Comment:
Is this injected Organization really needed? Seems it not used anywhere in added changes.